doc/mgr/administrator.rst

```diff
@@ -44,12 +44,12 @@ Calling module commands
 Where a module implements command line hooks, using the Ceph CLI's
 ``tell`` command to call them like this::
 
-    ceph tell mgr <command | help>
+    ceph tell mgr <command | get_command_descriptions>
```

Comment:
yes, `get_command_descriptions` is not meant for humans. I spent a lot of time to learn it. I have a simple idea. how about adding a new command `help` and just print all available commands?